CharlieBackend.AdminPanel/Views/Courses/AllCourses.cshtml

```diff
@@ -0,0 +1,261 @@
+﻿@model IEnumerable<CharlieBackend.AdminPanel.Models.Course.CourseViewModel>
+
+@{
+    Layout = "_Layout";
+}
+
+
+
+<div class="fixed-bottom ml-4" style="left:90%; right:50%; bottom:8%;">
+    <div class="dropleft">
+        <button type="button" class="btn btn-lg btn-success btn-circle " data-toggle="dropdown">
+            <i class="fas fa-plus"></i>
+        </button>
+        <div class="dropdown-menu ">
+            <form method="post" class="input-group" asp-action="AddCourse" asp-controller="Courses">
+                <input type="text" class="form-control" name="id" placeholder="Course id" />
+                <button type="submit" class="btn btn-outline-success btn-sm">
+                    <i class="fas fa-plus"></i>
+                </button>
+            </form>
```

Comment:
Since you're adding the button for "_add course_"
![qweqwe](https://user-images.githubusercontent.com/38729881/110213302-1eb95500-7ea8-11eb-857c-ee2cb080496b.png)
while not adding any `CoursesController.AddCourse()` method, you should probably also add some sort of placeholder or "to be added" message.

So we won't get a plain 404 when someone decides to try adding a new course:
![404](https://user-images.githubusercontent.com/38729881/110213418-9c7d6080-7ea8-11eb-8d76-8f385c50ad89.png)

Consider creating new issue if you're not going to add support for this feature in https://github.com/ita-social-projects/WhatBackend/issues/215

Reply:
Done. 4cf34647d2c0e0f87bd75be718150598293ca4e4
It's already has an issue https://github.com/ita-social-projects/WhatBackend/issues/449